test/popup.spec.js

```diff
@@ -154,18 +154,35 @@ describe('window.open', function() {
     expect(intercepted).toBe(true);
     await context.close();
   });
-  it('should apply addInitScript from browser context', async function({browser, server}) {
-    const context = await browser.newContext();
-    await context.addInitScript(() => window.injected = 123);
-    const page = await context.newPage();
-    await page.goto(server.EMPTY_PAGE);
-    const injected = await page.evaluate(() => {
-      const win = window.open('about:blank');
-      return win.injected;
+  describe('BrowserContext.addInitScript', function() {
```

Comment:
nit: the full name would be `window.open BrowserContext.addInitScript should apply to an in-process popup`. I'd rather make a nicer name without using describe.

Reply:
done